packages/flutter/lib/src/material/drawer.dart

```diff
@@ -4,6 +4,7 @@
 
 import 'package:flutter/foundation.dart';
 import 'package:flutter/widgets.dart';
+import 'package:flutter/material.dart';
```

Comment:
import the appropriate source files from material (like below) instead of the entire lib. The circular import will break the docs shard